shared/consts.go

```diff
@@ -14,4 +14,5 @@ const (
 	GoGCEnvVar                       = "GOGC"
 	DefaultApiServerPort             = 8899
 	DebugModeEnvVar                  = "MIZU_DEBUG"
+	DefaultDbPath                    = "./entries.db"
```

Comment:
The shared module is for objects that need to be shared between multiple modules (e.g. cli + agent), not for storing all consts.
`DefaultDbPath` is only used in agent.

Reply:
whoops